### DISCUSSION TITLE
switch account feature wanted

### DISCUSSION THREAD

**Discussion by xdewx** [community/community#39036] [Other Feature Feedback, Questions, & Ideas]
I have 2 accounts

**Comment by ghostinhershell** [comment]
Hey @xdewx 👋 

We're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails and what questions you have. Giving a few more details might help someone give you a nudge in the right direction. Cheers!

**Reply by DSDmark** [reply]
Ya, that right, anyway…

I also have 2 accounts 😆 .
